lib/timeline/timeline.dart

```diff
@@ -45,6 +46,17 @@ class TimelineScreen extends Screen {
 
   PButton pauseButton;
   PButton resumeButton;
+  PButton trackWidgetBuildsButton;
+  PButton perfOverlayButton;
+  PButton repaintRainbowButton;
+  PButton debugPaintButton;
+
+  final String trackWidgetBuildsServiceExtension =
```

Comment:
make these static

Reply:
Added a service_extensions.dart file to hold const ext names.